@ziegenberg just two failed unit tests, and they look pretty straightforward to me. Up to you on whether or not you want to spin up a dev environment for them.

```
=================================== FAILURES ===================================
______________________________ test_send_messages ______________________________
[gw0] linux -- Python 3.8.6 /var/lib/awx/venv/awx/bin/python3.8
Traceback (most recent call last):
  File "/awx_devel/awx/main/tests/unit/notifications/test_rocketchat.py", line 26, in test_send_messages
    requests_mock.post.assert_called_once_with('http://example.com', data='{"text": "test subject"}', verify=True)
  File "/usr/lib64/python3.8/unittest/mock.py", line 925, in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
  File "/usr/lib64/python3.8/unittest/mock.py", line 913, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: expected call not found.
Expected: post('http://example.com', data='{"text": "test subject"}', verify=True)
Actual: post('http://example.com', data='{"text": "test subject"}', headers={'Content-Type': 'application/json', 'User-Agent': 'AWX 0.1.dev1+ge8614f9 (open)'}, verify=True)
____________________ test_send_messages_with_no_verify_ssl _____________________
[gw0] linux -- Python 3.8.6 /var/lib/awx/venv/awx/bin/python3.8
Traceback (most recent call last):
  File "/awx_devel/awx/main/tests/unit/notifications/test_rocketchat.py", line 103, in test_send_messages_with_no_verify_ssl
    requests_mock.post.assert_called_once_with('http://example.com', data='{"text": "test subject"}', verify=False)
  File "/usr/lib64/python3.8/unittest/mock.py", line 925, in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
  File "/usr/lib64/python3.8/unittest/mock.py", line 913, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: expected call not found.
Expected: post('http://example.com', data='{"text": "test subject"}', verify=False)
Actual: post('http://example.com', data='{"text": "test subject"}', headers={'Content-Type': 'application/json', 'User-Agent': 'AWX 0.1.dev1+ge8614f9 (open)'}, verify=False)
```

--- a/awx/main/notifications/rocketchat_backend.py
+++ b/awx/main/notifications/rocketchat_backend.py
@@ -9,6 +9,7 @@ from django.utils.encoding import smart_text
 from django.utils.translation import ugettext_lazy as _
 
 from awx.main.notifications.base import AWXBaseEmailBackend
+from awx.main.utils import get_awx_http_client_headers
 from awx.main.notifications.custom_notification_base import CustomNotificationBase
 
 logger = logging.getLogger('awx.main.notifications.rocketchat_backend')
@@ -38,7 +39,9 @@ class RocketChatBackend(AWXBaseEmailBackend, CustomNotificationBase):
                 if optvalue is not None:
                     payload[optval] = optvalue.strip()
 
-            r = requests.post("{}".format(m.recipients()[0]), data=json.dumps(payload), verify=(not self.rocketchat_no_verify_ssl))
+            r = requests.post(
+                "{}".format(m.recipients()[0]), data=json.dumps(payload), headers=get_awx_http_client_headers(), verify=(not self.rocketchat_no_verify_ssl)
+            )
 
             if r.status_code >= 400:
                 logger.error(smart_text(_("Error sending notification rocket.chat: {}").format(r.status_code)))

--- a/awx/main/tests/unit/notifications/test_rocketchat.py
+++ b/awx/main/tests/unit/notifications/test_rocketchat.py
@@ -7,8 +7,11 @@ import awx.main.notifications.rocketchat_backend as rocketchat_backend
 
 
 def test_send_messages():
-    with mock.patch('awx.main.notifications.rocketchat_backend.requests') as requests_mock:
+    with mock.patch('awx.main.notifications.rocketchat_backend.requests') as requests_mock, mock.patch(
+        'awx.main.notifications.rocketchat_backend.get_awx_http_client_headers'
+    ) as version_mock:
         requests_mock.post.return_value.status_code = 201
+        version_mock.return_value = {'Content-Type': 'application/json', 'User-Agent': 'AWX 0.0.1.dev (open)'}
         backend = rocketchat_backend.RocketChatBackend()
         message = EmailMessage(
             'test subject',
@@ -23,7 +26,12 @@ def test_send_messages():
                 message,
             ]
         )
-        requests_mock.post.assert_called_once_with('http://example.com', data='{"text": "test subject"}', verify=True)
+        requests_mock.post.assert_called_once_with(
+            'http://example.com',
+            data='{"text": "test subject"}',
+            headers={'Content-Type': 'application/json', 'User-Agent': 'AWX 0.0.1.dev (open)'},
+            verify=True,
+        )
         assert sent_messages == 1
 
 
@@ -84,8 +92,11 @@ def test_send_messages_with_icon_url():
 
 
 def test_send_messages_with_no_verify_ssl():
-    with mock.patch('awx.main.notifications.rocketchat_backend.requests') as requests_mock:
+    with mock.patch('awx.main.notifications.rocketchat_backend.requests') as requests_mock, mock.patch(
+        'awx.main.notifications.rocketchat_backend.get_awx_http_client_headers'
+    ) as version_mock:
         requests_mock.post.return_value.status_code = 201
+        version_mock.return_value = {'Content-Type': 'application/json', 'User-Agent': 'AWX 0.0.1.dev (open)'}
         backend = rocketchat_backend.RocketChatBackend(rocketchat_no_verify_ssl=True)
         message = EmailMessage(
             'test subject',
@@ -100,5 +111,10 @@ def test_send_messages_with_no_verify_ssl():
                 message,
             ]
         )
-        requests_mock.post.assert_called_once_with('http://example.com', data='{"text": "test subject"}', verify=False)
+        requests_mock.post.assert_called_once_with(
+            'http://example.com',
+            data='{"text": "test subject"}',
+            headers={'Content-Type': 'application/json', 'User-Agent': 'AWX 0.0.1.dev (open)'},
+            verify=False,
+        )
         assert sent_messages == 1

